lnwallet/channel.go

```diff
@@ -7716,15 +7716,11 @@ func (lc *LightningChannel) ForceClose() (*LocalForceCloseSummary, error) {
 		return nil, err
 	}
 
-	localCommitment := lc.channelState.LocalCommitment
-	summary, err := NewLocalForceCloseSummary(
-		lc.channelState, lc.Signer, commitTx,
-		localCommitment.CommitHeight, lc.leafStore, lc.auxResolver,
+	// We don't need resolutions when force closing the channel so we call
```

Comment:
Tried this one but not sure if that is good design, probably not, because the caller does not necessary know that the resolutions are not set in the ChanArb ?